.github/workflows/generate-limits.yaml

```diff
@@ -0,0 +1,67 @@
+name: Generate Limits
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: "0 10 * * 1" # Every Monday at 10:00 UTC
```

Comment:
why every monday and not daily? is there some pre-determined schedule from aws side?

Reply:
No reason, we don't release CNI that frequently so started with weekly. I can make it more frequent